elide-graphql/src/main/java/com/yahoo/elide/graphql/GraphQLScalars.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2017, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.graphql;
+
+import graphql.language.IntValue;
+import graphql.language.StringValue;
+import graphql.schema.Coercing;
+import graphql.schema.CoercingParseValueException;
+import graphql.schema.GraphQLScalarType;
+import lombok.extern.slf4j.Slf4j;
+
+import java.time.Instant;
+import java.util.Date;
+
+/**
+ * Additional scalar/serializers for built-in graphql types.
+ */
+@Slf4j
+public class GraphQLScalars {
+    private static final String ERROR_BAD_EPOCH = "Date must be provided as epoch millis";
+    private static final String ERROR_BAD_EPOCH_TYPE = "Date must be provided as string or integral in epoch millis";
+
+    public static final GraphQLScalarType GRAPHQL_DATE_TYPE = new GraphQLScalarType(
```

Comment:
Are there tests for this class?

Reply:
No, this was the first pass. While I convert things internally to GraphQL I have been making my changes and pushing them to ensure I'm "doing the right thing." We'll definitely want tests for this before we merge.